addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -2169,7 +2169,7 @@ var FieldBooleanButton = AbstractField.extend({
 });
 
 var BooleanToggle = FieldBoolean.extend({
-    description: _lt("Toggle"),
+    description: _lt("Active"),
```

Comment:
This field is called `Toggle` for a reason, this value should not be changed unless we are really sure that we want to call that widget the `Active` widget.
What should be done, is simply to set a name on the field in the model, which will then be used as the value for this.

Which means that this https://github.com/odoo/odoo/blob/05d304851194ab1bbdd8af8e0d166091367b395d/addons/lunch/models/lunch_alert.py#L31
should be rewritten as that
```python
active = fields.Boolean("Active", default=True)
```